tasks/Carter Racing.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+Decision Making
+A small racing team is at a crossroads.  They have been experiencing engine failures all season, and are down to their last dollars, and are in danger of losing their sponsors.  However, when they did not suffer engine failure, their car and team were very successful.  They are within the final hours before the last big race of the season.  If they place in the top 5, which is very plausible if they choose to race, they will secure a large sponsor, Goodstone Tires.  If they are struck with another engine failure, they will lose their current sponsors and will not be able to continue racing next season.  What do they do?
+
+# References
+Brittain, J. & Sitkin, S. (1986). Carter racing. Case Study.
+
+Link: https://execed.poole.ncsu.edu/wp-content/uploads/2020/06/Carter-Racing-A_B_C.pdf
+
+Kray, Laura J., and Adam D. Galinsky. "The debiasing effect of counterfactual mind-sets: Increasing the search for disconfirmatory information in group decisions." Organizational Behavior and Human Decision Processes 91.1 (2003): 69-81.
+
+Link: https://www.researchgate.net/profile/Laura-Kray-2/publication/222830434_The_debiasing_effect_of_counterfactual_mind-sets_Increasing_the_search_for_disconfirmatory_information_in_group_decisions/links/5aa56ac60f7e9badd9ab57df/The-debiasing-effect-of-counterfactual-mind-sets-Increasing-the-search-for-disconfirmatory-information-in-group-decisions.pdf
+
+# Stimuli
+## The visual components
+The following figures are present in the case study
+
+![Alt text](/images/Carter_Racing_Exhibt_1.png)
+![Alt text](/images/Carter_Racing_Exhibt_C1.png)
+
```

Comment:
Pre-Task Description: 
> Counterfactual Condition: The protagonist Sue was at a rock concert of her favorite band. Because her seat was not very close to the stage, Sue switched to a vacant seat in the third row. Shortly thereafter, the emcee walked out on stage and announced that a valuable prize would be awarded to one lucky winner. The woman then reached her hand into a receptacle filled with ticket stubs and announced the winner was the person who currently occupied Sues old seat
> Non-Counterfactual Condition: . In the non-counterfactual prime condition, a similar scenario was presented, except that Sue did not switch seats and the lucky winner was someone in a different seat altogether.